request_workers.go

```diff
@@ -29,42 +29,65 @@ type RequestWorkers struct {
 type Worker struct {
 	Workers    chan chan *JobQItem
 	JobChannel chan *JobQItem
+	ctx        Context
 	id         int
 }
 
-func NewRequestWorkers(workers chan chan *JobQItem, id int) Worker {
+var workers []Worker
+
+func newRequestWorkers(workers chan chan *JobQItem, id int, ctx Context) Worker {
 	return Worker{
 		Workers:    workers,
 		JobChannel: make(chan *JobQItem),
+		ctx:        ctx,
 		id:         id,
 	}
 }
 
+// Process runs a goroutine for the worker
 func (pw Worker) Process(pubnub *PubNub) {
 	go func() {
+	B:
```

Comment:
Can we use a more descriptive label here?

Reply:
Modified to ProcessLabel